packages/web-app-admin-settings/src/views/Users.vue

```diff
@@ -156,34 +169,50 @@ export default defineComponent({
       return data
     })
 
+    const selectedPersonalDrives = reactive([])
     watch(
-      selectedUsers,
+      () => unref(selectedUsers).length,
       async () => {
-        const loadAdditionalData = unref(selectedUsers).length === 1
-        if (loadAdditionalData && unref(loadedUser)?.id === unref(selectedUsers)[0].id) {
-          // current user is already loaded
-          return
-        }
-
         sideBarLoading.value = true
-        if (loadAdditionalData) {
-          loadedUser.value = await loadAdditionalUserDataTask.perform(unref(selectedUsers)[0])
-          sideBarLoading.value = false
-          return
-        }
+        // Load additional user data
+        const requests = []
+        unref(selectedUsers).forEach((user) => {
+          requests.push(loadAdditionalUserDataTask.perform(user))
+        })
 
-        loadedUser.value = null
+        const loadedUsers = await Promise.all(requests)
+        unref(selectedUsers).forEach((user) => {
+          const additionalUserData = loadedUsers.find((loadedUser) => loadedUser.id === user.id)
+          Object.assign(user, additionalUserData)
+          if (unref(selectedUsers).length === 1) {
+            loadedUser.value = additionalUserData
+            return
+          }
+          loadedUser.value = null
```

Comment:
I think you need to move this out of the for-loop, because we also need to set `loadedUser.value = null` when no `selectedUsers` are given. Currently, when you select a user, open the sidebar and then de-select the user, `loadedUser ` stays the same.